Error in loading the page stops the script

This is bad for automatic runs, skip the run instead and log the error.

```
2022-03-16 08:18:30 [INFO ] Shutdown complete
2022-03-16 08:18:31 [INFO ] Found 0 new offers
2022-03-16 08:18:31 [INFO ] Waiting until 2022-03-16T08:38:31.313360 for next execution
2022-03-16 08:38:31 [INFO ] Starting Run # 335
2022-03-16 08:38:31 [INFO ] Start scraping of Amazon Prime
2022-03-16 08:38:31 [INFO ] Getting pagedriver options
2022-03-16 08:38:31 [INFO ] Creating driver
2022-03-16 08:38:31 [INFO ] Injecting JS
2022-03-16 08:40:39 [ERROR] Failure starting Chrome WebDriver, aborting: unknown error: net::ERR_CONNECTION_TIMED_OUT
  (Session info: headless chrome=99.0.4844.51)
```

--- a/lootscraper.py
+++ b/lootscraper.py
@@ -53,7 +53,12 @@ def main() -> None:
     run = 1
     while not exit.is_set():
         logging.info(f"Starting Run # {run}")
-        job()
+
+        try:
+            job()
+        except Exception as e:
+            # Something unexpected occurred, log it and continue with the next run as usual
+            logging.exception(e)
 
         time_between_runs = int(Config.config()["common"]["WaitBetweenRuns"])
         if time_between_runs == 0:

